include/silo/storage/column/pango_lineage_column.h

```diff
@@ -7,22 +7,27 @@
 #include <roaring/roaring.hh>
 
 #include "silo/common/pango_lineage.h"
+#include "silo/storage/column/column.h"
 
 namespace silo::storage::column {
 
-class PangoLineageColumn {
+class PangoLineageColumn : public Column {
   public:
    template <class Archive>
    [[maybe_unused]] void serialize(Archive& archive, const unsigned int /* version */) {
+      archive& values;
       archive& value_id_lookup;
+      archive& id_value_lookup;
       archive& indexed_values;
+      archive& indexed_sublineage_values;
    }
 
   private:
+   std::vector<uint32_t> values;
    std::unordered_map<common::PangoLineage, uint32_t> value_id_lookup;
+   std::vector<common::PangoLineage> id_value_lookup;
```

Comment:
How much performance benefit is there from using a `vector` over an `unordered_map`? We are essentially misusing the vector for a map. Using a map would make the code more obvious.

Reply:
I would think this is fine. Especially as it is the simpler data structure.